### ISSUE TITLE
Manual liabilities in the dashboard count only one record for each asset

### ISSUE THREAD

**Issue by evilny0** [rotki/rotki#3724]
In the dashboard manual liabilities section, there is one line for each asset, like in the assets section. That's what I expect since it's a summary.

What I did not expect: the amount displayed in each line is the amount of only one record per asset. I expect that line to display the sum of all liability records I entered in the manual balances, for this specific asset.

Records which are not counted are also ignored in the net balance.

I also noticed that the balance in the history graph seems to completely ignore manual liabilities, is that how it was designed ? I expected to see the net balance history in this graph.

Rotki version: 1.22.0

**Comment by LefterisJP** [comment]
> What I did not expect: the amount displayed in each line is the amount of only one record per asset. I expect that line to display the sum of all liability records I entered in the manual balances, for this specific asset.

Hello @evilny0. What do you mean? Each record's line should display its own line amount. That is normal. Can you maybe take a screenshot and explain what you see vs what you expect? You can use scrambling mode: https://rotki.readthedocs.io/en/latest/usage_guide.html#data-scrambling to make the data random.

> I also noticed that the balance in the history graph seems to completely ignore manual liabilities, is that how it was designed ? I expected to see the net balance history in this graph

What do you mean to ignore manual liabilities? They are subtracted from the net total value. That's what a liability is. A debt. So your net worth graph is what is affected.

**Comment by evilny0** [comment]
I installed rotki in a clean lxc container to take screenshots. Here is the manual liabilities list:

![image](https://user-images.githubusercontent.com/8269734/142183551-b8919fb9-273d-4f23-9a05-ca21d0b6c30d.png)

Here is the dashboard:

![image](https://user-images.githubusercontent.com/8269734/142183597-fbb19d49-3b12-486f-86b8-ec27e662db66.png)

See how the dashboard line for USDC is only 10.000 ? I expect it to be 25.000, or eventually to see two lines, like in the manual balances page.

For the history graph in the dashboard, I was wrong about the "ignore", it's more clear with the example amounts I added. It's actually adding the liabilities to the balance. I'll open a separate issue with screenshots.
